### DISCUSSION TITLE
[PvP] Daily PvP Battleground Rewards

### DISCUSSION THREAD

**Discussion by shaszun** [chromiecraft/chromiecraft#6046] [Suggestions]
Hi,
Since the arena release the PvP started to be more active again. Not sure if this was intended but the daily battleground rewards, doesn't give arena points and that's why I write it as suggestion here. This is like a little catch up way for people with low rating or just not liking arenas to get their points and buy some PvP set pieces and not wait for next season to be able to buy it with honor. Another thing I would suggest is to make the daily reward available for every day of the week, but maybe in reduced amount if it's not a weekend. This will make PvP active througout the whole week.

**Comment by balleny** [comment]
https://github.com/chromiecraft/chromiecraft/issues/6058
and
https://github.com/chromiecraft/chromiecraft/issues/5983
